met_connectors/src/lustre_netatmo/mod.rs

```diff
@@ -60,16 +60,16 @@ fn read_netatmo(timestamp: Timestamp) -> Result<DataCache, data_switch::Error> {
             lats.push(record.lat);
             lons.push(record.lon);
             elevs.push(record.elev);
-            values.push((
+            values.push(Timeseries {
```

Comment:
```suggestion
            data.push(Timeseries {
```
Can we rename it `data` for consistency?

Reply:
Done